tests/Cluster.py

```diff
@@ -856,7 +864,11 @@ def bootstrap(totalNodes, prodCount, biosHost, biosPort, dontKill=False, onlyBio
             Utils.Print("Wait for issue action transaction to become finalized.")
             transId=Node.getTransId(trans[1])
             # biosNode.waitForTransInBlock(transId)
-            biosNode.waitForTransFinalization(transId)
+            # guesstimating block finalization timeout. Two production rounds of 12 blocks per node, plus 60 seconds buffer
+            timeout = .5 * 12 * 2 * len(producerKeys) + 60
+            if not biosNode.waitForTransFinalization(transId, timeout=timeout):
```

Comment:
Since this looks like it has a print statement that would apply to waitForTransFinalization, I think adding reportNotFinalized flag and this print statement would make sense.

Reply:
Yes, this will also be added into a separate PR.